client/shipping-label/views/index.js

```diff
@@ -42,6 +62,7 @@ function mapStateToProps( state, { storeOptions } ) {
 	return {
 		shippingLabel: state.shippingLabel,
 		errors: getFormErrors( state, storeOptions ),
+		ratesTotal: getRatesTotal( state.shippingLabel.form.rates.values, state.shippingLabel.form.rates.available ),
```

Comment:
Added in https://github.com/Automattic/woocommerce-connect-client/pull/533/commits/8170c7f2a15a3f9542d4aad4e44b69526c5293ea.